docs/software-platforms/lsfr/creating-vcs.md

```diff
@@ -0,0 +1,131 @@
+# Creating virtual computers with Lightsail for Research
+
+While working on an [analysis module](../../contributing-to-analyses/analysis-modules/index.md), you can develop locally or on a virtual computer with Lightsail for Research (LSfR).
+
+Virtual computers provide access to a set amount of virtual CPUs (vCPUs), memory, and storage through a web browser rather than a physical machine.
+Using a virtual computer allows you to develop and run analyses that require more CPUs and memory than is available on your local machine.
+
+!!! note
+
+    You will need access to AWS to use LSfR.
+    See [getting access to AWS](../../getting-started/accessing-resources/index.md#getting-access-to-aws) to learn how to get set up with AWS.
+
+- The virtual computers provided through LSfR use an Ubuntu operating system.
+- During setup, you will choose your desired configuration, including the amount of virtual CPUs, memory, and storage space.
+    - You also have the option to add [additional storage to your virtual computer](STUB-LINK for volumes).
+- All virtual computers will have a set of pre-installed applications needed for working with OpenScPCA, including:
+    - Git and GitKraken
+    - The AWS command line interface, which has already been configured for you
+    - R
+    - Conda
+
+
+!!! tip "More on virtual computers"
+
+    Read more about [virtual computers with Lightsail for Research](https://docs.aws.amazon.com/lightsail-for-research/latest/ug/computers.html).
+
+## How to create a virtual computer
+
+Follow the below steps to create a virtual computer to use with LSfR.
+
+1. Navigate to the [access portal URL from when you set up your user in IAM Identity center](../aws/index.md#joining-iam-identity-center).
+This link should bring you to the AWS Console and home page.
```

Comment:
This is missing a step, I think – don't you need to select the account and then click on `ResearcherRestriction` to get into the console?